src/helpers/alert.ts

```diff
@@ -12,3 +14,10 @@ export async function generalFailureAlert(message: string) {
 export async function generalSuccessAlert(message: string) {
   return sweetAlert('Success', message, 'success')
 }
+
+export async function transferFailureAlert(failedTxPayload: any) {
+  const txErrorCode = get(failedTxPayload, 'data.extras.result_codes.transaction', 'unknown_error')
+  const opErrorCode = get(failedTxPayload, 'data.extras.result_codes.operations[0]', 'unknown_error')
+  const errorMessage = transactionErrorCodeToMessage(txErrorCode, opErrorCode)
```

Comment:
It may be good to move the above into a separate function since it could be called elsewhere potentially in the future with sweet alert.